### PR TITLE
Show agreements on expenses & revamp additional info

### PR DESCRIPTION
**Pull request by Betree**
Resolve https://github.com/opencollective/opencollective/issues/6705
~~Resolve https://github.com/opencollective/opencollective/issues/6831~~

Also includes some standardization of font-sizes.

image

### CODE REVIEW

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--bank (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--other (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--pay-pal (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--bank (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--other (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--pay-pal (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--bank (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--other (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--pay-pal (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--bank (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--other (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--pay-pal (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--bank (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--other (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--pay-pal (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

File: components/expenses/ExpenseForm.js [hunk 1]
```diff
@@ -749,7 +749,7 @@ const ExpenseFormBody = ({
       )}
       {step === EXPENSE_FORM_STEPS.EXPENSE && (
         <StyledCard mt={4} p={[16, 24, 32]} overflow="initial">
-          <ExpensePayeeDetails expense={formik.values} host={collective.host} borderless collective={collective} />
+          <ExpenseSummaryAdditionalInformation expense={formik.values} host={collective.host} collective={collective} />
```
Comment (Betree) [inline components/expenses/ExpenseForm.js:752]: Renamed the component as the legacy name was misleading, this holds more info than just the payee details

File: components/expenses/ExpenseSummaryAdditionalInformation.js [hunk 2]
```diff
@@ -123,8 +131,63 @@ const ExpensePayeeDetails = ({ expense, host, isLoading, borderless, isLoadingLo
...
+          <PrivateInfoColumnHeader>{formatCollectiveType(intl, collective.type)}</PrivateInfoColumnHeader>
+          <LinkCollective collective={collective}>
+            <Flex alignItems="center">
+              <Avatar collective={collective} radius={24} />
+              <Flex flexDirection="column" ml={2} mr={2} css={{ overflow: 'hidden' }}>
+                <Span color="black.800" fontSize="14px" fontWeight="700" truncateOverflow>
+                  {formatAccountName(collective.name, collective.legalName)}
+                </Span>
+                <Span color="black.900" fontSize="12px" truncateOverflow>
+                  @{collective.slug}
```
Comment (Betree) [inline components/expenses/ExpenseSummaryAdditionalInformation.js:147]: Showing the handles is a small deviation in design (I informed them about it), but the handle is an essential detail for making sure it's the right profile; it makes it more difficult to impersonate someone

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--bank (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--other (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--pay-pal (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--bank (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--other (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--pay-pal (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--bank (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--other (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--pay-pal (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 6  | ❗       |
    


<details>
<summary>❗ storybook/expenses-expense-summary--invoice (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--loading (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-expense-summary--receipt (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--bank (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--other (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/expenses-payout-method-data--pay-pal (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Pull request merged**